auto_round/utils.py

```diff
@@ -318,6 +318,22 @@ def validate_modules(module_names, quant_vision=False, vison_blocks_names=None):
                          "or raise an issue at https://github.com/intel/auto-round/issues.")
     return
 
+def get_common_prefix(paths):
+    # Split each path into components and find the common prefix
+    split_paths = [path.split('.') for path in paths]
+    common_prefix = split_paths[0]
+    for path in split_paths[1:]:
+        common_prefix = [comp for comp, other in zip(common_prefix, path) if comp == other]
+    return '.'.join(common_prefix)
+
+def extract_block_names_to_str(quant_block_list):
+    if not isinstance(quant_block_list, (list,tuple)):
+        return None
+    # Extract common prefix for each list
+    prefixes = [get_common_prefix(blocks) for blocks in quant_block_list]
```

Comment:
should have a double check if the common perfix include some other layers, it have, should set it to 16 bits